### PR TITLE
Pythonic Syntax for __getattribute__ Error.

### PR DESCRIPTION
**Pull request by ansaboutin**
Added a python_path function to format a __getattribute__ runtime error to python syntax.

### CODE REVIEW

File: src/ansys/fluent/core/solver/flobject.py [hunk 1]
```diff
@@ -159,6 +170,20 @@ def path(self) -> str:
...
+
+        Constructed in python syntax from 'python_path' and the parents python
+        path.
+        """
+        if self._parent is None:
+            return "<session>"
+        ppath = self._parent.python_path
+        if not ppath:
+            return self.python_name
+        return ppath + self.python_name
```
Comment (mkundu1) [inline src/ansys/fluent/core/solver/flobject.py:185]:
```suggestion
        return ppath + "." + self.python_name
```

File: src/ansys/fluent/core/solver/flobject.py [hunk 2]
```diff
@@ -145,6 +145,17 @@ def obj_name(self) -> str:
...
+    @property
+    def python_name(self) -> str:
+        """Python name of this object.
+
+        By default, this returns the object's static name. If the object is a child of a
+        named object, the object's name is returned.
+        """
+        if self._name:
+            return self._python_name
+        return "." + self.__class__.__name__
```
Comment (mkundu1) [inline src/ansys/fluent/core/solver/flobject.py:157]:
```suggestion
        return getattr(self, "_python_name") or self.__class__.__name__
```

File: src/ansys/fluent/core/solver/flobject.py (hunk 1 above)
Comment (ansaboutin) [reply src/ansys/fluent/core/solver/flobject.py:184]: I did undo this change as I was getting a "." before the bracket:
image

File: src/ansys/fluent/core/solver/flobject.py [hunk 3]
```diff
@@ -145,6 +146,15 @@ def obj_name(self) -> str:
             return self.fluent_name
         return self._name
 
+    @property
+    def python_name(self) -> str:
+        """Python name of this object.
+
+        By default, this returns the object's static name. If the object is a child of a
+        named object, the object's name is returned.
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/solver/flobject.py:154]: But we can also return the class name prefixed with a dot. That doesn't look right.

**Review by dnwillia-work — APPROVED** [review]

**Review by mkundu1 — APPROVED** [review]

**Review by prmukherj — APPROVED** [review]

**Pull request merged**